dynamiqs/utils/tensor_types.py

```diff
@@ -130,14 +129,16 @@ def dtype_real_to_complex(
 
 
 @complex_tensor
-def from_qutip(x: Qobj, *, dtype=None, device=None) -> Tensor:
+def from_qutip(
+    x: Qobj, *, dtype: torch.dtype | None = None, device: torch.device | None = None
```

Comment:
In the case where we use `@complex_tensor` decorator you want to use`dtype: torch.complex64 | torch.complex128 | None = None`. BUt it's quite length, this is why we removed the type in a PR a while ago for functions decorated with `@complex_tensor`. We can do as you prefer, either fix the typing everywhere, or remove the typing everywhere and just specify that it takes a complex data type in the dosctring description of `dtype`.

Reply:
No I had to had it back because it was not appearing in the documentation actually. If you remove it and try to build to doc locally, you'll see that type hints are gone.